dxr/static_unhashed/css/dxr.css

```diff
@@ -190,13 +188,19 @@ code a {
     background-color: transparent;
 }
 
-/* results */
+/* promoted stuff, on the top */
+.promoted-container {
+    padding: 0.05em 1.0em 0;
```

Comment:
I dropped the border because it overlapped the switch tree menu. And now I realize I could avoid it by using a margin instead.